core/commands/show_file_at_commit.py

```diff
@@ -0,0 +1,39 @@
+import sublime
+from sublime_plugin import WindowCommand, TextCommand
+
+from ..git_command import GitCommand
+
+
+SHOW_COMMIT_TITLE = "COMMIT: {}"
+
+class GsShowFileAtCommitCommand(WindowCommand, GitCommand):
+
+    def run(self, commit_hash, filepath):
+        self.filepath = filepath
+        repo_path = self.repo_path
+        currentview = self.window.active_view()
+        lang = currentview.settings().get('syntax')
+        view = self.window.new_file()
+        view.set_syntax_file(lang)
+        view.settings().set("commit_hash", commit_hash)
+        view.settings().set("filepath", filepath)
+        view.settings().set("git_savvy.repo_path", repo_path)
+        view.settings().set("word_wrap", False)
+        view.settings().set("line_numbers", True)
+        view.set_name(SHOW_COMMIT_TITLE.format(commit_hash[:7]))
```

Comment:
One other little thing.  Let's indicate which file is being viewed, as well as the revision.  Something like this.
```python
SHOW_COMMIT_TITLE = "{}:{}"

# ...
    def run(self, commit_hash, filepath):
# ...
        view.set_name(SHOW_COMMIT_TITLE.format(commit_hash[:7], filepath)
```

 replace `commit_hash[:7]` here with `